docassemble/AssemblyLine/al_document.py

```diff
@@ -1360,7 +1408,7 @@ def send_button_html(
         the bundle to the specified email address.
 
         Optionally, display a checkbox that allows someone to decide whether or not to
-        include an editable (Word) copy of the file, iff it is available.
+        include an editable (Word) copy of the file, if it is available.
```

Comment:
```suggestion
        include an editable (Word) copy of the file, if and only if it is available.
```
FYI, maybe I will avoid this shorthand but iff = if and only if--not the same as "if"